### PR TITLE
Tweak and add schema from 18f/data-act-schemas

### PR DESCRIPTION
**Pull request by micahsaul**
Changed the schema to nest objects instead of referencing them by ID.
This uses "Action" as the most granular object. A new image will be
forthcoming.

### CODE REVIEW

File: schema/data-act-schema.proto [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    optional string periodOfPerformanceStart = 13;
+    optional string periodOfPerformanceCurrentEnd = 14;
+    optional string periodOfPerformancePotentialEnd = 15;
+    optional string orderingPeriodEndDate = 16;
+    repeated PlaceOfPerformance placeOfPerformance = 17;
+    repeated Awardee awardee = 18;
+    optional string NAICSCode = 19;
+    optional string NAICSDescription = 20;
+    optional string CFDAProgramNumber = 21;
+    optional string CFDAProgramType = 22;
```
Comment (kaitlin) [inline schema/data-act-schema.proto]: I think this should be CFDAProgramTitle?

**Comment by micahsaul** [comment]
D'oh! Yes, it should. It was wrong on the old diagram, as well.

**Comment by micahsaul** [comment]
Also, I need to add Award Type or whatever we want to call it. What do we want to call it?

**Comment by kaitlin** [comment]
hmmm I keep seeing inconsistent names for this in the doc. I see both Transaction Type Code and Award Type... I think I need to verify with Eileen and/or Bryan. Let's go with Type of Transaction Code since that's the field listed in the public data elements, but make a note that it doesn't really make sense right now :)

**Pull request merged**